src/NuGet.Core/NuGet.Resolver/NuGet.Resolver.csproj

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <Description>NuGet's dependency resolver within the NuGet.Packaging package.</Description>
-    <TargetFrameworks>$(TargetFrameworksLibrary)</TargetFrameworks>
+    <TargetFrameworks>$(TargetFrameworksLibrary);netstandard2.1</TargetFrameworks>
```

Comment:
Update the property in the common targets.

Reply:
Thanks! Changed accordingly.